catalyst/engines/__init__.py

```diff
@@ -13,9 +13,21 @@
 
 if SETTINGS.amp_required:
     from catalyst.engines.amp import AMPEngine, DistributedDataParallelAMPEngine
+else:
+    # replacement
+    from catalyst.engines.device import DeviceEngine as AMPEngine
+    from catalyst.engines.distributed import (
+        DistributedDataParallelEngine as DistributedDataParallelAMPEngine,
+    )
```

Comment:
why do we need it? let's do not mess the instances of the engines
it could become unreproducible in this case

Reply:
Sure, I'll remove "replacement" imports from engines